runner/src/mill/runner/MillMain.scala

```diff
@@ -326,16 +326,15 @@ object MillMain {
 
         val bspServerHandleRes = {
           val wsRoot = WorkspaceRoot.workspaceRoot
-          mill.bsp.BspClasspathWorker(wsRoot).flatMap { worker =>
-            val logDir = wsRoot / OutFiles.out / "mill-bsp"
-            os.makeDir.all(logDir)
-            worker.startBspServer(
-              define.WorkspaceRoot.workspaceRoot,
-              splitStreams,
-              logDir,
-              true
-            )
-          }
+          val worker = mill.bsp.BspClasspathWorker(wsRoot, logger = ???)
```

Comment:
Actually, this shouldn't be merged for now because of this (added while rebasing)

Reply:
Should be fixed